src/components/Search/SearchInput.tsx

```diff
@@ -61,6 +62,12 @@ const SearchInput = ({ onStateChange }: SearchInputListener) => {
 
   const resultCount = searchItems.length;
 
+  useEffect(() => {
+    if (currentFeedbackType) {
+      setcurrentFeedbackType(undefined);
+    }
+  }, [searchValue]);
```

Comment:
This change is necessary as when the results list opens/closes - it resets the state of our feedback widget. Therefore we will track if feedback has been sent or not via the parent.